### ISSUE TITLE
Convert shapefiles to geojson

### ISSUE THREAD

**Issue by arnfinn** [Helseatlas/shinymap#23]
Not recommended to use shapefiles

**Comment by arnfinn** [comment]
Step by step, taken from [blog.exploratory.io](https://blog.exploratory.io/creating-geojson-out-of-shapefile-in-r-40bc0005857d):

1. Read shapefile

```R
map <- rgdal::readOGR(dsn = ...)
```

2. Convert to geojson

```R
geojson_map <- geojsonio::geojson_json(map)
```

3. Reduce size

```R
geojson_map_reduced <- rmapshaper::ms_simplify(geojson_map, keep = 0.1) # "keep" is optional

4. Write to geojson file

```R
geojsonio::geojson_write(geojson_map_reduced , file = "<path>/map.geojson")
```

**Comment by arnfinn** [comment]
Fixed in #25
